Mathlib/Analysis/SpecialFunctions/Trigonometric/Basic.lean

```diff
@@ -503,6 +503,23 @@ theorem cos_eq_sqrt_one_sub_sin_sq {x : ℝ} (hl : -(π / 2) ≤ x) (hu : x ≤
   rw [← abs_cos_eq_sqrt_one_sub_sin_sq, abs_of_nonneg (cos_nonneg_of_mem_Icc ⟨hl, hu⟩)]
 #align real.cos_eq_sqrt_one_sub_sin_sq Real.cos_eq_sqrt_one_sub_sin_sq
 
+lemma cos_half {x : ℝ} (hl : -π ≤ x) (hr : x ≤ π) : cos (x / 2) = sqrt ((1 + cos x) / 2) := by
```

Comment:
```suggestion
lemma cos_div_two {x : ℝ} (hl : -π ≤ x) (hr : x ≤ π) : cos (x / 2) = sqrt ((1 + cos x) / 2) := by
```
seems better aligned with the naming scheme (and below)

Reply:
(unless there's already precedent that I've missed)

Reply:
Precedents: https://loogle.lean-lang.org/?q=%22half%22 https://loogle.lean-lang.org/?q=%22halves%22